### ISSUE TITLE
request implementation of conv1DLSTM, conv3DLSTM in keras

### ISSUE THREAD

**Issue by zh794390558** [tensorflow/tensorflow#23393]
**System information**
- 1.12
- No

**Describe the feature and the current behavior/state.**
there is no conv1DLSTM, conv3DLSTM in keras.

**Will this change the current api? How?**
no

**Comment by tensorflowbutler** [comment]
Nagging Assignee @fchollet: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @fchollet: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by rmothukuru** [comment]
@zh794390558,
Sorry for the delayed response.

Can you please let us know the use cases for the new APIs that you have requested? Thanks!
